docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md

```diff
@@ -79,7 +79,12 @@ az account list --query "[?isDefault]"
 
 # get a subscription that contains search words or phrases
 az account list --query "[?contains(name,'search phrase')].{SubscriptionName:name, SubscriptionID:id, TenantID:tenantId}" --output table
+```
+
+---
+# [Bash](#tab/bash)
```

Comment:
Before we start a tab group, we need to tell the reader what we're doing, like "You can also store subscription information in a variable for use within a script."

Reply:
Noted for future PRs as well.